sdk/spring-sdk-testkit/src/test/java/springsdk/testmodels/actions/EchoActionTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package springsdk.testmodels.actions;
+
+import kalix.javasdk.testkit.ActionResult;
+import kalix.springsdk.testkit.ActionTestkit;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class EchoActionTest {
+
+  @Test
+  public void testEchoCall() {
+    ActionTestkit<EchoAction> actionUnitTestkit = ActionTestkit.of(ctx -> new EchoAction());
+    ActionResult<String> result = actionUnitTestkit.call(echoAction -> echoAction.echo("Hey"));
+    Assertions.assertEquals(result.getReply(), "Hey");
+  }
```

Comment:
We don't need extensive tests. We are already relying on the existing test infra.